packages/katello/katello/helper.rb

```diff
@@ -32,8 +32,16 @@ def error_message
       "This utility can't run on a non-katello system."
     end
 
+    def scenario_yaml(scenario)
+      File.join(scenarios_path, "#{scenario}.yaml")
+    end
```

Comment:
Nit: Above are a bunch of paths. Should this be together with `last_scenario_yaml` (which should also use `File.join()` IMHO, but we're getting deep into to "why did I look" territory).?

Reply:
Can certainly move it up there.